File: factcheck.js
function httpGet(theUrl)
{
    var xmlHttp = new XMLHttpRequest();
    xmlHttp.open( "GET", "https://factchecktools.googleapis.com/v1alpha1/claims:search" , false ); // false for synchronous request
    xmlHttp.send();
    return xmlHttp.responseText;
}

File: factcheck.py
def httpGetFactCheck(text):
	response = requests.get('https://factchecktools.googleapis.com/v1alpha1/claims:search?query='+str(text)+'?languageCode=en-US')
	dic = json.loads(response.text) 

	print(response.status_code)
	if not response:
		print("Error in Fact Check")
	else:
		print (dic)
		rval = dic['claimReview'] # this is a list
		print (len(rval))
		return rval[0]['textualRating']
return None



